cicd/3-app/javabuilder/config/adhoc.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Parameters": {
+    "BaseDomainName": "code.org",
+    "BaseDomainNameHostedZonedID": "Z2LCOI49SCXUGU",
+    "ProvisionedConcurrentExecutions": "1",
+    "ReservedConcurrentExecutions": "3",
+    "LimitPerHour": "15",
+    "LimitPerDay": "40",
+    "SilenceAlerts": "false"
```

Comment:
also if adhocs are on prod we should set SilenceAlerts to true. Anything on the dev account gets silenced by default in the template